sdk/communication/Azure.Communication.NetworkTraversal/src/Models/GetRelayConfigurationOptions.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Communication.NetworkTraversal.Models
+{
+    /// <summary> GetRelayConfigurationOptions. </summary>
+    public class GetRelayConfigurationOptions
+    {
+        /// <summary> The <see cref="CommunicationUserIdentifier"/> for whom to issue a token. </summary>
+        public CommunicationUserIdentifier CommunicationUser { get; set; }
+
+        /// <summary> The specified <see cref="RouteType"/> for the relay request. </summary>
+        public RouteType? RouteType { get; set; }
+
+        /// <summary> Initializes a new instance of GetRelayConfigurationOptions. </summary>
+        public GetRelayConfigurationOptions()
```

Comment:
This can be deleted, the default constructor gets automatically generated by C#.